docs/semgrep-sc/sc-getting-started.md

```diff
@@ -28,12 +29,14 @@ Object.entries(frontMatter).filter(
 }
 </ul>
 
-# Scanning open source dependencies 
+# Getting started with Semgrep Supply Chain
 
-Semgrep Supply chain scans can be triggered from the Semgrep App interface.
+Semgrep Supply Chain detects recently discovered [security vulnerabilities](https://nvd.nist.gov/vuln/full-listing) through [reachability](/docs/semgrep-sc/sc-glossary#reachability) analysis. 
```

Comment:
detecting via reachability analysis might be misleading, maybe can be something like 

"SSC detects recently discovered security vulnerability in your code, prioritizing urgent ones via reachability analysis"

Reply:
Added! Thank yeew. Merging.